ruby/spec/gherkin/gherkin_line_spec.rb

```diff
@@ -32,5 +32,9 @@ def cells_text(line)
     it 'does not drop white spaces inside a cell' do
       expect(cells_text("| foo()\n  bar\nbaz |")).to eq(["foo()\n  bar\nbaz"])
     end
+
+    it 'trailing escapes are ignored' do
+      expect(cells_text("| a | b \\")).to eq(['a'])
```

Comment:
should this be 2 separate specs given it's also testing that items outside of the table are ignored (Or is this in a diff spec somewhere else?)